jwst/lib/suffix.py

```diff
@@ -89,7 +89,7 @@
     'saturation',
     'pathloss',
     'groupscalestep',
-    'rampfit',
+    'ramp_fit',
```

Comment:
Hmmm, so it looks like stpipe was already using a suffix of "ramp_fit", because it's the one creating the output files. Not sure why it was using that name already, but I guess we'll just go with it.